versioned/storage/jdbc-tests/src/main/java/org/projectnessie/versioned/storage/jdbctests/CockroachBackendTestFactory.java

```diff
@@ -30,6 +30,7 @@ public String getName() {
   @Nonnull
   @Override
   protected JdbcDatabaseContainer<?> createContainer() {
-    return new CockroachContainer(dockerImage("cockroach"));
+    return new CockroachContainer(
+        dockerImage("cockroach").asCompatibleSubstituteFor("cockroachdb/cockroach"));
```

Comment:
What motivates the addition of `.asCompatibleSubstituteFor()`? I've seen it added in a few places, but not everywhere.

Reply:
Required by the testcontainer classes.